test/integration/test_summarization_accuracy_semantic_robustness.py

```diff
@@ -43,98 +41,90 @@
 )
 
 
-class TestCaseEvaluate(NamedTuple):
+class TestCase(NamedTuple):
     config: SummarizationAccuracySemanticRobustnessConfig
-    expected_evaluate_sample_scores: Dict[str, float]
-    expected_evaluate_scores: Dict[str, float]
+    evaluate_sample_scores: Dict[str, float]
+    evaluate_scores: Dict[str, float]
 
 
 class TestSummarizationAccuracySemanticRobustness:
     @pytest.mark.parametrize(
-        "config, expected_evaluate_sample_scores, expected_evaluate_scores",
+        "config, evaluate_sample_scores, evaluate_scores",
```

Comment:
Note that as with GSR, I have verified that the SASR logic remains the same as before by temporarily using the old numpy APIs in the semantic perturbation code, and then running the test cases using the old expected scores.